next/components/forms/simple-components/ServiceCard.tsx

```diff
@@ -0,0 +1,50 @@
+import cx from 'classnames'
+import Link from 'next/link'
+import { FunctionComponent, SVGProps } from 'react'
+
+import CircleArrowRight from '../../../assets/images/circle-arrow-right.svg'
+
+type ServiceCardBase = {
+  title: string
+  description: string
+  buttonText: string
+  Logo: FunctionComponent<SVGProps<SVGSVGElement>>
+  href: string
+  className?: string
+}
+
+const ServiceCard = ({
+  title,
+  description,
+  buttonText,
+  Logo,
+  href,
+  className,
+}: ServiceCardBase) => {
+  const style = cx(
+    'group min-w-[280px] max-w-[280px] bg-gray-0 border-gray-200 flex flex-col items-start p-5 gap-5 border-solid border-2 rounded-lg cursor-pointer',
+    className,
+  )
+
+  return (
+    <Link href={href}>
```

Comment:
we still did not understand each other - even the Link should not be part of the component itself. Just unwrap it from this, we'll be adding either Link or `<a>` tag from the parent component later when we'll be using the cards